frontend/pages/packs/AllPacksPage/AllPacksPage.jsx

```diff
@@ -225,29 +224,29 @@ export class AllPacksPage extends Component {
 
       return (
         <div>
-          <p className={`${baseClass}__pack-count`}>
-            {checkedPackCount} {packText} Selected
-          </p>
+          <span className={`${baseClass}__pack-count`}>
+            <strong>{checkedPackCount}</strong> selected
+          </span>
           <Button
             className={`${btnClass} ${btnClass}--disable`}
             onClick={onBulkAction("disable")}
             variant="unstyled"
```

Comment:
```suggestion
            variant="text-icon"
```
We're starting to introduce some inconsistency in when we choose to use certain button variants. Can you please edit these buttons to use the existing `text-icon` button variant?

 The text-icon button variant styles can be found in the `/Button/_styles.scss` file here: https://github.com/fleetdm/fleet/blob/875d6bb6acc4f99afe0d6cdae0760b956c1e1245/frontend/components/buttons/Button/_styles.scss#L145-L173

An example use of the `text-icon` button variant can be found here: https://github.com/fleetdm/fleet/blob/875d6bb6acc4f99afe0d6cdae0760b956c1e1245/frontend/pages/admin/TeamManagementPage/TeamDetailsWrapper/TeamDetailsWrapper.tsx#L176-L181

An additional bonus is that the `<img />` used for the icons will be in the same `.jsx` file as the button. We also get to use the `alt` tag.